plugin/hooks.go

```diff
@@ -155,4 +156,8 @@ type Hooks interface {
 	// Note that this method will be called for files uploaded by plugins, including the plugin that uploaded the post.
 	// FileInfo.Size will be automatically set properly if you modify the file.
 	FileWillBeUploaded(c *Context, info *model.FileInfo, file io.Reader, output io.Writer) (*model.FileInfo, string)
+
+	// NotificationWillBeSent is invoked before a notification is sent.
+	// To modify the notified user map, return the replacement map,
```

Comment:
Please add a minimum server version
```suggestion
	// To modify the notified user map, return the replacement map,
	//
	// Minimum server version: 5.20
```